packages/view/backend/sirius-components-view-emf/src/main/java/org/eclipse/sirius/components/view/emf/diagram/StylesFactory.java

```diff
@@ -110,6 +112,19 @@ public String getNodeType(NodeStyleDescription nodeStyleDescription) {
         return type.orElse(NodeType.NODE_RECTANGLE);
     }
 
+    public DiagramStyle createDiagramStyle(DiagramStyleDescription diagramStyleDescription) {
+        if (diagramStyleDescription == null) {
+            return null;
+        }
```

Comment:
Why would this method be called with `null`? I would prefer not have have this kind of defensive programming and fix the issue where it is being created.